Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.mm

```diff
@@ -250,6 +251,8 @@
     LOG_WITH_STREAM(RemoteLayerTree, stream << "RemoteLayerTreeDrawingAreaProxy::commitLayerTree transaction:" << layerTreeTransaction.description());
     LOG_WITH_STREAM(RemoteLayerTree, stream << "RemoteLayerTreeDrawingAreaProxy::commitLayerTree scrolling tree:" << scrollingTreeTransaction.description());
 
+    const CFTimeInterval beginTime = CFAbsoluteTimeGetCurrent();
```

Comment:
Simon suggested a C++ RAII class to handle this bookending for you, and I think he was right.

Reply:
Said class would also obviate the need for the indirection that we added in RemoteScrollingTree.